webapp/components/utils/InternalPages.js

```diff
@@ -0,0 +1,180 @@
+function noStringDefined(string) {
+  return !string || string === 0
+}
```

Comment:
Can this be a private function within the class below please?

Reply:
This function I moved now in the class below.

Thanks for the hint, Ulf. 😍